independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/providers/serialisers/FileBodyHandler.java

```diff
@@ -21,8 +21,8 @@
 import org.jboss.resteasy.reactive.common.headers.HeaderUtil;
 
 public class FileBodyHandler implements MessageBodyReader<File>, MessageBodyWriter<File> {
```

Comment:
Is this even used outside of the Quarkus version, which now overrides it? If not, I wonder if it wouldn't be less complex to ditch the reader overrides build items and pass the proper config and cleaner to the non-quarkus RR, no?

Reply:
🤷